cicd/3-app/javabuilder/integration-test-buildspec.yml

```diff
@@ -0,0 +1,28 @@
+version: 0.2
+phases:
+  install:
+    runtime-versions:
+      nodejs: 14
+  pre_build:
+    commands:
+      - node -v
+      - npm -v
+      - cd ./integration-tests
+      - npm install
+  build:
+    commands:
+      - set -e
+
+      - # TODO: Make the ARN and/or secret names configurable too?
+      - private_key=$(aws secretsmanager get-secret-value --secret-id arn:aws:secretsmanager:us-east-1:475661607190:secret:development/cdo/javabuilder_integration_tests_key-b4BQG2)
+      - password=$(aws secretsmanager get-secret-value --secret-id arn:aws:secretsmanager:us-east-1:475661607190:secret:development/cdo/javabuilder_integration_tests_key_password-sWpRnW)
```

Comment:
Could we replace the IDs here with `${AWS::AccountId}`? Thought it would only work on the production AWS account currently (we'd need to add the integration test keys to the dev account if we want it to work in dev too).